crev-lib/src/local.rs

```diff
@@ -281,8 +281,30 @@ impl Local {
     }
 
     pub fn init_readme_using_this_repo_file(&self) -> Result<()> {
+        const README_MARKER_V0: &str = "CREV_README_MARKER_V0";
+
         let proof_dir = self.get_proofs_dir_path()?;
-        std::fs::write(proof_dir.join("README.md"), &include_bytes!("../rc/doc/README.md")[..])?;
+        let path = proof_dir.join("README.md");
+        if path.exists() {
```

Comment:
I suggest that it would be beneficial to move this `if` statement into it's own function: `repo_readme_exists(.)`, for example. Easier to handle future readme markers. But not a big deal at all.